.rubocop.yml

```diff
@@ -0,0 +1,258 @@
+AllCops:
+  TargetRubyVersion: 2.6
+  Exclude:
+    - 'lib/resources/example_json/*.rb'
```

Comment:
let's lint the example json files too

Reply:
I want to be clear, if we actually lint the example JSON files this will require us to fix almost an additional 10,000 lint errors. (that's not an exaggeration). This is _primarily_ because of the `Style/StringLiterals` linter (preferring single quotes strings), but there's also the `Layout/LineLength` style as well. My primary reason for disabling this was because of the amount of changes and we mostly copy and paste the examples from sandbox and don't alter them in any way. If you want me to I can enable it, but the changes this will introduce will certainly outweigh the rest of this PR. 

It might be better if the lint errors specifically for the examples are updated and resolved (with the disabling removed) in this PR: https://github.com/cerner/fhir.cerner.com/pull/420 since that is updating the examples already.

Reply:
Logged issue: https://github.com/cerner/fhir.cerner.com/issues/423